test/web3.eth.getSyncing.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+var chai = require('chai');
+var Web3 = require('../index');
```

Comment:
Thanks. But this will change in 1.0 I am working right now on the `modularize` branch where I'm fixing right now all tests and adopt them anyway